tests/09-mdbook-mode/input/.gitignore

```diff
@@ -0,0 +1 @@
+book
```

Comment:
Why do we need this?

Reply:
It is created by `mdbook library`

Reply:
Why should mdbook create a file? It should only read the files na?

Reply:
No it is creating, `.gitignore`, `book.toml` and a `book` directory, If they are not present.

Reply:
We can even remove `mdbook` dependency and construct `toc` by tree only.

Reply:
We can `create_gitignore` to `false` to not to create it.

Reply:
Ok, we are expecting other than ft-sync.p1 there is nothing else to be created. Our binary creating other files would be an issue. See if you can suppress it somehow. Else we can go ahead for now, note it as a TODO, and get this backend working asap.